qcodes/dataset/sqlite/queries.py

```diff
@@ -305,7 +306,7 @@ def get_parameter_data_for_one_paramtree(
     # faster than transposing the data using np.array.transpose
     res_t = map(list, zip(*data))
 
-    for paramspec, column_data in zip(paramspecs, res_t):
+    for paramspec, column_data in zip_longest(paramspecs, res_t, fillvalue=[]):
```

Comment:
probably does not matter, but should the filler be immutable for safety/sanity?
```suggestion
    for paramspec, column_data in zip_longest(paramspecs, res_t, fillvalue=tuple()):
```

Reply:
Sure, I cannot see how this would matter but there is no harm